index.html

```diff
@@ -33,6 +33,14 @@ <h1 class="header-content">Discover hidden places in the world around you</h1>
                     <source src="./resources/excursion.mp4" type="video/mp4" />
                 </video>
             </section>
+
+            <section>
+                <p class="main-personal-guide">Your personal travel guide</p>
+                <p class="main-remember-place">
+                    Excursion remembers places you like, and recommendeds new pointes of interest around you.
+                </p>
+                <img src="./resources/camp.jpg" alt="camp-image" />
```

Comment:
I this it should be inside the div container for adjusting the image size in future.